lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2128,8 +2128,75 @@ TypeSystemSwiftTypeRef::GetByteStride(opaque_compiler_type_t type,
 
 lldb::Encoding TypeSystemSwiftTypeRef::GetEncoding(opaque_compiler_type_t type,
                                                    uint64_t &count) {
-  return m_swift_ast_context->GetEncoding(ReconstructType(type), count);
+  if (!type)
```

Comment:
This should probably also be inside impl?

Reply:
In the other PRs I've done, I placed this check outside of the `impl`, since they're trivial. Should they be moved into the `impl`?

Reply:
I would say yes, because we'd want to VALIDATE even the trivial error case, right?

Reply:
done